plugins/python/python.py

```diff
@@ -346,15 +380,35 @@ def list_plugins():
         lib.hexchat_print(lib.ph, b'No python modules loaded')
         return
 
-    lib.hexchat_print(lib.ph, b'Name         Version  Filename             Description')
-    lib.hexchat_print(lib.ph, b'----         -------  --------             -----------')
+    lib.hexchat_print(
```

Comment:
I think this mostly hinders readability.

Reply:
Yeah, Im not 100% sure I like that, it is required to keep it within PEP8 line length limits

Reply:
changed in later commit